client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-action-bar.scss

```diff
@@ -0,0 +1,45 @@
+.pattern-assembler {
+	.pattern-action-bar {
+		display: flex;
+		opacity: 0;
+		align-items: center;
+		position: absolute;
+		z-index: 1;
+		background: #fff;
+		height: 27px;
+		right: 0;
+
+		.pattern-action-bar__block {
+			flex-direction: column;
+			display: flex;
+			align-items: center;
+		}
+
+		.pattern-action-bar__action {
```

Comment:
[nit] Could we increment the hit box for the close button? 
![Screenshot 2023-03-08 at 9 14 07 AM](https://user-images.githubusercontent.com/797888/223594534-07059f3f-814d-410c-b197-335f33da437a.png)

Reply:
Sure!